boards/arm/bl5340_dvk/Kconfig

```diff
@@ -13,6 +13,13 @@ config MBOX_NRFX_IPC
 config RPMSG_SERVICE_DUAL_IPM_SUPPORT
 	default RPMSG_SERVICE
 
+if IPC_SERVICE_BACKEND_RPMSG
+
+config IPC_SERVICE_BACKEND_RPMSG_SHMEM_RESET
+	default y
+
+endif # IPC_SERVICE_BACKEND_RPMSG
```

Comment:
Just wondering, why this is defined in each board including nRF5340, instead of nrf5340 SoC?

Reply:
I observed that the configuration of other IPC modules (RPMsg service dual-mode) is defined in the boards Kconfig, so I assumed that there is some implicit agreement to have configuration there.

I can move this part to the SoC Kconfig to avoid duplications.

Reply:
I don't know this implicit agreement. Just curios why these duplications are needed. @carlocaione , do you know?

Reply:
IIRC this was due to a limitation of the legacy IPC services (like rpmsg service) but that should not be needed anymore with  IPC service so this can be moved to SoC level.

Reply:
I moved the configuration from boards Kconfig to SoC Kconfig. Please take a look